main.cpp

```diff
@@ -465,39 +529,46 @@ struct TerminalBuilding
      wasAnythingDetected);
 };
 
+TerminalBuilding::TerminalBuilding()
+{
+    std::cout << "TerminalBuilding being constructed!" << std::endl;
+}
+
 float TerminalBuilding::chargeBaggageFee(float numOfBags, float costPerBag)
 {
      float baggageFee = numOfBags * costPerBag;
+     std::cout << "\nBaggage Fee amount: $ " << baggageFee << std::endl;
      return baggageFee;  
 }
 
 void TerminalBuilding::purchaseTicket(bool isTicketPaid)
 {
     if (isTicketPaid)
     {
-        std::cout << "Ticket purchased" << std::endl;
+        std::cout << "Ticket purchased." << std::endl;
     }
 }
 
 void TerminalBuilding::screenPassengersForSecurity(bool didPassengerWalkThorughImageDetector, bool wasAnythingDetected)
 {
     if ((didPassengerWalkThorughImageDetector == true) && (wasAnythingDetected == false))
     {
-        std::cout << "Please continue walking" << std::endl;
+        std::cout << "Please continue walking." << std::endl;
     }
     else if ((didPassengerWalkThorughImageDetector == true) && (wasAnythingDetected == 
     true))
```

Comment:
Don't split expressions like this onto multiple lines.
split at the `&&`:
```
if( expr1 &&
    expr2 &&
    expr3 )
{
    statements();
}
```
- [ ] revise

Reply:
I need to start fixing these spacing issues with main.cpp on full screen instead of console and main displays up together on one screen. Thanks.